app/src/main/java/org/oppia/app/parser/StringToFractionParser.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.app.parser
+
+import org.oppia.app.model.Fraction
+
+/** This class contains method that helps to parse string to fraction. */
+class StringToFractionParser {
+  public fun getFractionFromString(text: String): Fraction {
```

Comment:
`Optional Nit`: This entire function is correct but the code is not much readable. Can we make this more eye-friendly?

Reply:
Updated